app/components/have-a-question.js

```diff
@@ -0,0 +1,131 @@
+import React, { useState, useEffect, useRef } from 'react'
+import { createUseStyles } from 'react-jss'
+import TextareaAutosize from 'react-textarea-autosize'
+import cx from 'classnames'
+
+const defaultValue = 'Have a question? Ask away....'
```

Comment:
this const name should be more descriptive of it's purpose - how about placeHoderMessage
It shouldn't be default - because that implies it could be updated.